### PR TITLE
Update manual deployment with extra info

### PR DESCRIPTION
**Pull request by surfer190**
I just went through the manual deployment steps and would like to add a few things:

* The `libsecp256k1-dev` requirement
* Using a virtual environment
* Info about using a process manager

### CODE REVIEW

File: deployment/manual.md [hunk 1]
```diff
@@ -78,6 +87,11 @@ sudo pip3 install -r requirements/daemons/coin_name.txt
 
 Where coin\_name is coin code(btc, ltc, etc.).
 
+Optional: Create virtual environment instead of using the global python environment
+
+    python3.9 -m venv env
```
Comment (MrNaif2018) [inline deployment/manual.md]:
```suggestion
    python3 -m venv env
```
More generic version

File: deployment/manual.md [hunk 2]
```diff
@@ -145,16 +159,27 @@ python3 worker.py
...
+cd bitcart-store
 NUXT_PORT=4000 yarn start
 ```
 
-Your BitcartCC API will run on port 8000, daemons on ports 5000-500X, admin panel on 3000, store on 4000.
+#### Default ports
+
+* The BitcartCC API runs on port `8000`.
+* Daemons on ports `5000-500X`
+* The bitcartCC Admin panel runs on port `3000`
```
Comment (MrNaif2018) [inline deployment/manual.md]:
```suggestion
* The BitcartCC Admin panel runs on port `3000`
```

File: deployment/manual.md [hunk 3]
```diff
@@ -145,16 +159,27 @@ python3 worker.py
...
+* THe BitcartCC Store runs on port `4000`.
```
Comment (MrNaif2018) [inline deployment/manual.md]:
```suggestion
* The BitcartCC Store runs on port `4000`.
```

**Review by MrNaif2018 — CHANGES_REQUESTED** [review]
Wow, thank you very much for those additions! Small review comments

**Comment by surfer190** [comment]
Will do. There are a few more things I must also add.

**Comment by MrNaif2018** [comment]
Oh wow! Incredible additions! By the way, is it really possible to specify env vars you set by separating them with `;` ?

As for firewall setup, yeah. 
Well, doesn't bitcart discover merchants API via the default http://localhost:8000 URL? Or do you mean that it works when in local browser, but if you let's say have a server then it won't be accessible? Hmm, well, I used manual deployment method as something close to development setup but with production builds.

**Comment by surfer190** [comment]
> is it really possible to specify env vars you set by separating them with ; ?

No it isn't, they must be seperated by a space. My mistake. I corrected it now.

> Or do you mean that it works when in local browser, but if you let's say have a server then it won't be accessible?

Yes works perfect locally but I am setting it up on a remote VM and accessing it over the internet. Should I make that more clear on the docs?

Next step is setting up a wallet to get a `xpub` - must I do this with [electrum](https://electrum.org/#home)?

**Comment by MrNaif2018** [comment]
Well, by the way, why did you pick up manual deployment and not docker one? Personal preferences, or to try how it works?

No, you can get xpub with any wallet you want. It might not be an xpub, depends on what chain do you want to use

And yeah, please make it more clear that it's only when you set it up on a VM, if you set it up locally it will work just fine

**Comment by MrNaif2018** [comment]
Because since the last customizations, docker system is so customizable that manual deployment is almost never used

**Review by MrNaif2018 — APPROVED** [review]
Thank you very much! Merging!

**Pull request merged**